app/assets/stylesheets/users.scss

```diff
@@ -59,6 +59,10 @@
   }
 }
 
+.primary-checkpoint-remember-me {
```

Comment:
Same issue is happening in other pages as well in firefox, Eg in assignment page. 

We need to fix the component instead of applying a bandaid.

Reply:
@tachyons thanks. I'll be fixing this.